src/status_im2/contexts/wallet/create_account/view.cljs

```diff
@@ -93,8 +95,9 @@
         [quo/color-picker
          {:default-selected @account-color
           :on-change        #(reset! account-color %)
-          :container-style  {:padding-horizontal 12
-                             :padding-vertical   12}}]]
+          :window-width     window-width
```

Comment:
imo this prop should just be `width`. 
also can we consider just defaulting this inside the component so we can overwrite it but we don't need to pass this each use as it's common for all screens using this? 🤔

Reply:
Done